sourcerer-test-utils/src/main/java/org/elder/sourcerer/eventstore/test/rules/EventStoreRule.java

```diff
@@ -0,0 +1,75 @@
+package org.elder.sourcerer.eventstore.test.rules;
+
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import com.shazam.tocker.AliveStrategies;
+import com.shazam.tocker.DockerInstance;
+import com.shazam.tocker.PortMap;
+import com.shazam.tocker.RunningDockerInstance;
+import org.junit.rules.ExternalResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * JUnit rule for starting and stopping an EventStore instance using docker.
+ */
+public class EventStoreRule extends ExternalResource {
+    private static final String SERVICE_HOSTNAME = "localhost";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "SERVICE_HOSTNAME" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.elder.org/coding_rules#rule_key=squid%3AS1068)